README.md

```diff
@@ -154,6 +154,9 @@ This is a library for package management, and ensures a smoother experience than
 ```
 python terminal.py
 ```
+10. (Optional) Speeding up opening process in the future
```

Comment:
May be worth pointing out that this is for Windows only. Correct?

Reply:
Yes, good point. I'll change the readme to reflect that, and if I get the chance, I'll try to create a similar program for Mac OS to speed up the opening process

Reply:
Nice sounds good!!